### PR TITLE
use string wrap in semgrep_output_v1.atd part 3, fpath

### PR DESCRIPTION
**Pull request by aryx**
test plan:
make e2e

### CODE REVIEW

File: src/osemgrep/cli_scan/Output.ml [hunk 1]
```diff
@@ -43,7 +44,7 @@ let apply_fixes (conf : Scan_CLI.conf) (cli_output : Out.cli_output) =
   let edits : Textedit.t list =
     Common.map_filter
       (fun (result : Out.cli_match) ->
-        let path = result.Out.path in
+        let path = !!(result.Out.path) in
```
Comment (emjin) [inline src/osemgrep/cli_scan/Output.ml:47]: I'm not a huge fan of mysterious operators like this one. It makes the codebase harder to onboard onto (in fact as a reviewer I don't actually know for sure what it does, I just guessed). Not expecting us to change it now, just mentioning it

File: src/osemgrep/language_server/requests/Code_actions.ml [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
  * LICENSE for more details.
  *)
-
+open File.Operators
```
Comment (emjin) [inline src/osemgrep/language_server/requests/Code_actions.ml:15]: 🎉 Glad we're starting to break up Common

**Review by emjin — APPROVED** [review]

File: src/osemgrep/cli_scan/Output.ml (hunk 1 above)
Comment (aryx) [reply src/osemgrep/cli_scan/Output.ml:47]: Not a fan in general, but I have to admit this one, which was designed by @mjambon, is really useful to migrate code from using Common.filename to Fpath.t. Without it the refactoring is really annoying, and in the long term everything will be an fpath so you will need far less of those !!

Comment (emjin) [reply src/osemgrep/cli_scan/Output.ml:47]: Thanks for the context! That makes sense

**Pull request merged**